frontend/simple/views/Invite.vue

```diff
@@ -21,31 +88,50 @@
                 </a>
               </p>
           </div>
+          <i18n v-if="error" id="badUsername" class="help is-danger">Invalid Username</i18n>
+          <i18n v-if="self" class="help is-danger">Cannot Invite Yourself</i18n>
```

Comment:
It is really confusing in the code when I see `this.self = true` what that means.

Could you eliminate `this.self` completely and replace it with something like `this.errorMsg = L('Description of error')` and then have just one line above that displays whatever the error is?